legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldRewriter.java

```diff
@@ -122,7 +123,23 @@ public boolean visit(SQLIdentifierExpr expr) {
         String fullFieldName = arr[1];
 
         String index = curScope().getAliases().get(alias);
+        if (index == null) {
```

Comment:
I don't see issue related. Could you elaborate what's the current behavior without the changes? From what I see here, we try to find the index corresponding to an alias. If not found, does it mean the alias may be wrong?

Reply:
The issue is that it doesn't get caught as a specific error so the response is unclear/unactionable -- when you pass invalid indices, this index not being found propagates up as
```
OpenSearchException[Internal error occurred while processing request]
For more details, please send request for Json format to see the raw response from OpenSearch engine.
```